README.md

```diff
@@ -16,6 +16,15 @@ CLIENT_REPO_ROOT=${PWD}/c
 # Install pre-requisites
 sudo apt-get install libssl-dev libcurl4-openssl-dev uncrustify libyaml-dev
 
+# Build pre-requisite: libwebsockets
```

Comment:
Do we want to build from source? Or just apt-get install libwebsockets-dev?

Reply:
We need build from source for libwebsockets now.

Because the code in this PR uses the latest stable version `v4.2`, but the version of `libwebsockets-dev` in Ubuntu 20.04 LTS (docker image `ubuntu-latest`) is `v3.2.1`, which is not compatible.

Users using Ubuntu 20.10 or above can install libwebsockets-dev `v4.x` by 
```shell
apt-get install libwebsockets-dev
```

We can update it after `Ubuntu 22.04 LTS` releases next year.